spec/views/hq/partners/_form.html.erb_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+require 'cgi'
+
+RSpec.describe "hq/partners/_form.html.erb", type: :view do
+  before :each do
+    @provinces = Province.all
+    @statuses = Status.all
+    @partner = build_stubbed :partner
+  end
+
+  context 'render the form and show data in form controls' do
```

Comment:
`context 'renders value' do
  example 'Name' do`
This seems more clear in the code, test descriptions, and object heirarchy